ams/optimizer/src/main/java/com/netease/arctic/optimizer/LocalOptimizerContainer.java

```diff
@@ -40,7 +44,8 @@ protected Map<String, String> doScaleOut(String startUpArgs) {
     try {
       String[] cmd = {"/bin/sh", "-c", startUpArgs};
       LOG.info("Starting local optimizer using command:" + startUpArgs);
-      runtime.exec(cmd);
+      Process process = runtime.exec(cmd);
+      printProcessErrorStream(process.getErrorStream());
```

Comment:
Tools such as IOUtils.readLines(process.getErrorStream()).stream().collect(Collectors.joining("\n")) can be used to obtain information about the exceptions.

Reply:
I feel that your suggestion looks very good.